.github/workflows/release.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         uses: docker/build-push-action@v4
         with:
           context: .
-          file: ./docker/Dockerfile.keyprovider
+          file: ./attestation-agent/docker/Dockerfile.keyprovider
```

Comment:
I think this Dockerfile.keyprovider should be fixed; currently it has this:

```
COPY . .
RUN cd coco_keyprovider && cargo install --path .
```
but this probably won't work if we copy _only_ the `attestation-agent` dir, because it wouldn't be able to find `../image-rs` etc.

Reply:
@dubek thanks for the catch. Pushed a fix.